src/components/DataCollector/validateUserEventOptions.js

```diff
@@ -35,6 +35,7 @@ export default ({ options }) => {
     data: objectOf({}),
     documentUnloading: boolean(),
     renderDecisions: boolean(),
+    decisionContext: objectOf({}),
```

Comment:
Could you please provide more details why this is needed?

New options related to personalization should go under `personalization` block.